stable/kubewatch/Chart.yaml

```diff
@@ -1,15 +1,18 @@
 name: kubewatch
-version: 0.3.1
+version: 0.4.0
 apiVersion: v1
-appVersion: v0.0.3
-home: https://github.com/skippbox/kubewatch
+appVersion: v0.0.4
+home: https://github.com/bitnami-labs/kubewatch
 description: Kubewatch notifies your slack rooms when changes to your cluster occur
 keywords:
 - kubernetes
 - slack
+- hipchat
+- mattermost
+- flock
 maintainers:
 - name: compleatang
   email: casey@monax.io
```

Comment:
@tompizmor didn't know Owners files were a thing in this repo, meant adding yourself here in Chart.yaml. but happy either way.

Reply:
Yes, you can see more info about OWNERS files in [this post](https://codeengineered.com/blog/2017/kubernetes-chart-owners/) from Matt Farina:

However, apart from creating this OWNERS file, I could add Bitnami as maintainer too. Check the [Moodle chart](https://github.com/kubernetes/charts/blob/master/stable/moodle/Chart.yaml#L15-L16) for example.
We have some automated process around this and we would be able to automatically send PR to update the chart for every new release of Kubeapps.  

Example of automatic PR - https://github.com/kubernetes/charts/pull/5636

If you are OK with that I will update the PR!

Reply:
:100: OK with that.